generators/spa/templates/VueJS/Module.csproj

```diff
@@ -49,14 +49,14 @@
   </PropertyGroup>
   <Import Project="..\..\Build\SolutionReferences.targets" />
   <ItemGroup>
-    <PackageReference Include="DotNetNuke.Core" Version="9.8.0" />
-    <PackageReference Include="DotNetNuke.Instrumentation" Version="9.8.0" />
-    <PackageReference Include="DotNetNuke.Web" Version="9.8.0" />
-    <PackageReference Include="DotNetNuke.Web.Client" Version="9.8.0" />
-    <PackageReference Include="DotNetNuke.Web.Deprecated" Version="9.8.0" />
-    <PackageReference Include="DotNetNuke.WebApi" Version="9.8.0" />
+    <PackageReference Include="DotNetNuke.Core" Version="9.11.0" />
```

Comment:
@ArielBlanco1990 There should be a variable to use for this, instead of hard-coding the version number here. Please follow the pattern that was in place before this update. It previously used `dnnBuildVersionShort`. If that doesn't work and there's not a `dnnBuildVersionLong` or something, then we should have it created. Ideally, this update should be done in as few places as possible when we need to do it again in the future.

Reply:
@WillStrohl I followed your instructions and solved this problem. There was no `dnnBuildVersionLong `but there was `dnnBuildVersion `which is what we need. I replaced `<%= dnnBuildVersionShort %>` and `<%= dnnBuildVersion %>` in each case.